public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -39,15 +41,17 @@ export const LogsPanel: React.FunctionComponent<LogsPanelProps> = ({
     );
   }
 
-  const spacing = css`
-    margin-bottom: ${theme.spacing(1.5)};
-    //We can remove this hot-fix when we fix panel menu with no title overflowing top of all panels
-    margin-top: ${theme.spacing(!title ? 2.5 : 0)};
-  `;
-
   return (
     <CustomScrollbar autoHide>
-      <div className={spacing}>
+      <div className={style.container}>
+        {showCommonLabels && (
+          <div className={style.labelContainer}>
+            <span className={style.label}>Common labels:</span>
+            <LogLabels
+              labels={commonLabels ? (commonLabels.value as Labels) : ({ labels: '(no common labels)' } as Labels)}
```

Comment:
```suggestion
              labels={commonLabels ? (commonLabels.value as Labels) : ({ labels: '(no common labels)' })}
```
```suggestion
              labels={commonLabels ? (commonLabels.value as Labels) : ({ labels: '(no common labels)' } as Labels)}
```

i think the second `as Labels` is not necessary (well, i removed it and typescript did not complain)